tests/virt/node/high_performance_vm/test_numa.py

```diff
@@ -9,40 +9,10 @@
     get_vm_cpu_list,
 )
 from utilities.constants import SRIOV
-from utilities.infra import ExecCommandOnPod
 from utilities.network import sriov_network_dict
 from utilities.virt import VirtualMachineForTests, fedora_vm_body
 
-pytestmark = [
-    pytest.mark.usefixtures(
-        "skip_if_workers_vms",
-        "skip_if_no_cpumanager_workers",
-        "skip_if_numa_not_configured_or_enabled",
-    ),
-]
-
-
-@pytest.fixture(scope="module")
-def skip_if_numa_not_configured_or_enabled(schedulable_nodes, workers_utility_pods):
-    cat_cmd = "cat /etc/kubernetes/kubelet.conf"
-    single_numa_node_cmd = f"{cat_cmd} | grep -i single-numa-node"
-    topology_manager_cmd = f"{cat_cmd} | grep -w TopologyManager"
-    for cmd in (single_numa_node_cmd, topology_manager_cmd):
-        if not check_numa_config_on_node(
-            cmd=cmd,
-            schedulable_nodes=schedulable_nodes,
-            utility_pods=workers_utility_pods,
-        ):
-            pytest.skip(f"Test should run on nodes with {cmd.split()[-1]}")
```

Comment:
instead of removing skip - how about failing the test run with clear message that numa is not available?

Reply:
+1, this will help to debug if, for example, we will mistakenly run with `numa` marker, while numa is not configured

Reply:
added check back + `fail`